components/constellation/constellation.rs

```diff
@@ -1680,14 +1682,12 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
                 if let Err(e) = result {
                     self.handle_send_error(parent_pipeline_id, e);
                 }
-                Some(source_id)
+                None
```

Comment:
I'm not very familiar with webdriver, but its the only user of the pipeline id returned from this method. Does this change break anything there?